### DISCUSSION TITLE
Wi-Fi Issues

### DISCUSSION THREAD

**Discussion by paulknox** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#610] [Q&A]
I have a V1 board running without issues however more often than not the Wi-Fi signal is dropping due to distance. I have added a Wi-Fi antenna which helped a little bit it's still really hit and miss connecting... any ideas on how best to improve signal. Hot tub is in garden under decking and about 7 meters away from a Wi-Fi mesh beacon in the house. House is solid brick / concrete.

Thanks

Paul

**Comment by visualapproach** [comment]
There is a lot of DIY Wi-Fi booster antennas on YT. Never tried myself.

**Reply by paulknox** [reply]
Thanks... that's what I have added. Wondered if there was a way to increase the timeout on the server though as when I ping I sometimes get long delays? ![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/836850/3a8f2f5d-562a-4601-9ff0-91e55ef692d0)

**Reply by visualapproach** [reply]
possibly, but you have to compile the project yourself in that case.
What's the RSSI? Is the webserver working as expected or does it only load pages partly? In that case try updating the firmware.

**Reply by paulknox** [reply]
RSSI is about -80. Sometimes the pages load fine and other times it just times out.... running latest beta and updated today

![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/836850/0faffc22-d540-413f-9487-814d2ce47234)

**Reply by visualapproach** [reply]
Yes that's a bit weak I guess. Have you tried other Wi-Fi channels?

**Reply by paulknox** [reply]
Unfortunately I'm running a mesh network using tp link and this does not allow manual switching. I'm wondering if an external access point might be the best solution...
